frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -462,23 +463,32 @@ function doHealthPing(
 
   xhr.timeout = timeoutMs
 
+  function getFriendlyCommandLine(): string {
```

Comment:
Add comment describing what this method does and why it's needed.

Reply:
removed.